roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -107,30 +107,27 @@ def get_var(self, *keys, **kwargs):
                 return kwargs["default"]
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
 
-        convert = kwargs.get("type")
+        convert = kwargs.get("convert", lambda x: x)
```

Comment:
There are cases where the user can pass in JSON - disk availability for instance... but it might be best anyway to be explicit about expected values for those.